pkgs/games/dxx-rebirth/default.nix

```
@@ -22,13 +23,13 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "dxx-rebirth";
-  version = "unstable-2023-03-23";
+  version = "unstable-2024-01-13";
```

```suggestion
  version = "0-unstable-2024-01-13";
```

According to more recent contributing guide.